README.md

```diff
@@ -89,10 +76,64 @@ Then, visit http://127.0.0.1:8000/. This page displays a "Congratulations!" mess
   - Fully integrated with Django's authentication framework.  
   - Supports native user management features like creating users and sessions.
 
+## Limitations of django-mongodb-backend
 
-- **Management Commands**  
-    
-  - Use commands like `migrate`, `makemigrations`, `flush`, `sqlmigrate`, many more.
+- Database Variables `ATOMIC_REQUESTS`, `AUTOCOMMIT`, `CONN_HEALTH_CHECKS`, `TIME_ZONE` not supported
+- No support for GeoDjango
+- Functions such as `Chr`, `ExtractQuarter`, `MD5`, `Now`, `Ord`, `Pad`, `Repeat`, `Reverse`, `Right`, `SHA1`, `SHA224`, `SHA256`, `SHA384`, `SHA512`, and `Sign`.
+- The `tzinfo` parameter of the `Trunc` database functions does not work properly because MongoDB converts the result back to UTC.
+- Schema Validation is not enforced. Refer to MongoDB documentation for how to enforce schema validation.
+- Django DDL Transactions are not supported.
+- The `migrate --fake-initial` command is not supported due to the inability to introspect MongoDB collection schema.
+- The asynchronous functionality of the Django API has not yet been tested.
+- `BSONRegExp` has no custom field class. It is best represented as a `CharField`.
+
+
+#### **Model Limitations**
+
+  - `$vectorSearch` and `$search` and Geospatial index creation through the Django Indexes API is not yet available.
+  - Updating indexes in `EmbeddedModels` do not work after the first table creation.
+
+  - **ArrayField**
+    - Does not support `EmbeddedModel` within `ArrayField`.
+
+  - **EmbeddedModel**
+    - Limited schema change support (no changing of embedded models).
+    - Embedded documents cannot take Django ForeignKeys.
+    - Arbitrary or untyped `EmbeddedModelField` is not supported. All fields must derive from an `EmbeddedModel` class.
```

Comment:
There is no expectation of this though.

Reply:
This is a common pattern for MongoDB users, though.